src/js/experimental/media-chrome-listbox.js

```diff
@@ -136,6 +157,8 @@ class MediaChromeListbox extends globalThis.HTMLElement {
     if (!indicator)
       indicator = this.shadowRoot.querySelector(`[name="${name}-indicator"] > svg`);
 
+    // @ts-ignore
+    indicator = indicator.cloneNode(true);
```

Comment:
question(non-blocking): 
it looks like we mean two things by `indicator` here:
1. a "reference element" for the indicator (is this guaranteed to always exist? looks like yes, but just confirming)
2. the instance that will be used for a given option, which is a clone of (1) and gets some updates based on its usage.

We may want to either rename this method to clarify or split this into two different methods/functions for clarity.

Reply:
good call, removing the slot attribute was causing issues if it was done on the original element.
I'll improve this